### DISCUSSION TITLE
How to sort/alter order of attributes based on options/params.

### DISCUSSION THREAD

**Discussion by marvfinsy** [okuramasafumi/alba#355] [Q&A] [answered]
Need way to filter attributes returned and modify order of attributes. I can see how to possibly do filtering but see no way to alter order of attributes.

**Comment by okuramasafumi** [comment, marked as answer]
@marvfinsy 
For filtering, you can see https://github.com/okuramasafumi/alba?tab=readme-ov-file#filtering-attributes.
For modifying the order of attributes, I believe you can override `attributes` instance method like below:

```ruby
class SomeResource
  include Alba::Resource
  def attributes
    super.sort # Or sort as you like
  end
end
```

**Reply by marvfinsy** [reply]
will give it a go... thx
